packages/react-components/src/components/AppFrame/components/NavigationTopBar/NavigationTopBar.tsx

```diff
@@ -0,0 +1,140 @@
+import * as React from 'react';
+
+import { Close } from '@livechat/design-system-icons';
+import cx from 'clsx';
+
+import { Button } from '../../../Button';
+import { Icon } from '../../../Icon';
+import { useAppFrameAnimations } from '../../hooks/useAppFrameAnimations';
+
+import { INavigationTopBarProps, ITopBarAlertProps } from './types';
+
+import styles from './NavigationTopBar.module.scss';
+const baseClass = 'navigation-top-bar';
+const alertClass = `${baseClass}__alert`;
+
+/**
+ * NavigationTopBar is used to display a top bar in the app frame - mostly to display alerts, but arbitrary content can be placed there too.
+ * @example
+ * <NavigationTopBar>
+ *     <NavigationTopBar.Alert kind="error">
+ *       Something went wrong
+ *     </NavigationTopBar.Alert>
+ * </NavigationTopBar>
+ */
+export const NavigationTopBar = ({
+  children,
+}: INavigationTopBarProps): React.ReactElement => {
+  return <div className={styles[baseClass]}>{children}</div>;
+};
+
+/**
+ * Alert component for the NavigationTopBar.
+ * @example
+ * <NavigationTopBar>
+ *    <NavigationTopBar.Alert kind="error" show={!!error}>
+ *     Something went wrong
+ *   </NavigationTopBar.Alert>
+ * </NavigationTopBar>
+ * @example
+ * <NavigationTopBar>
+ *     <NavigationTopBarAlert
+ *      kind="warning"
+ *      primaryCta={{ label: 'Save', onClick: () => console.log('Save clicked') }}
+ *      secondaryCta={{ label: 'Discard', onClick: () => console.log('Discard clicked') }}
+ *      closeButton={{ onClick: () => console.log('Close clicked') }}
+ *      show={!hasUnsavedChanges && !hasClosedAlert}
+ *      >
+ *      You have unsaved changes
+ *     </NavigationTopBarAlert>
+ * </NavigationTopBar>
+ **/
+export const NavigationTopBarAlert: React.FC<ITopBarAlertProps> = ({
+  className,
+  kind = 'info',
+  closeButton,
+  children,
+  primaryCta,
+  secondaryCta,
+  show = true,
+}) => {
+  const alertRef = React.useRef<HTMLDivElement>(null);
+  const { isMounted, isOpen } = useAppFrameAnimations({
```

Comment:
While this hook isn't specifically designed for this component, it does work

Reply:
This hook is designed for `AppFrame`, and top bar is a part of the whole component, so I don't see any problems with using it here :)